File: ASP.NET Core/GraphQL_API/Models/Dvd.cs
using System;
using System.Collections.Generic;

namespace GraphQL_API.Models
{
    public partial class Dvd
    {
        public Dvd()
        {
            Dvdlanguage = new HashSet<Dvdlanguage>();
        }

        public long? DvdId { get; set; }
        public long MovieId { get; set; }
        public string Isbn { get; set; }
        public string Edition { get; set; }
        public string Region { get; set; }

        public virtual Movie Movie { get; set; }
        public virtual ICollection<Dvdlanguage> Dvdlanguage { get; set; }
    }
}

File: Express.js/models/DVD.js
/* jshint indent: 2 */

module.exports = function(sequelize, DataTypes) {
  const DVD = sequelize.define('DVD', {
    dvd_id: {
      type: DataTypes.INTEGER,
      allowNull: true,
      primaryKey: true
    },
    movie_id: {
      type: DataTypes.INTEGER,
      allowNull: false
    },
    isbn: {
      type: DataTypes.TEXT,
      allowNull: false
    },
    edition: {
      type: DataTypes.TEXT,
      allowNull: true
    },
    region: {
      type: DataTypes.TEXT,
      allowNull: true
    }
  }, {
    tableName: 'DVD',
    timestamps: false
  });

  DVD.associate = function(models) {
    models.DVD.belongsTo(models.Movie,
      { foreignKey: "movie_id" });
  };

  return DVD;
};
